prime-router/src/main/kotlin/Report.kt

```diff
@@ -36,6 +36,7 @@ class Report {
 
     /**
      * The sources that generated this service
+     * todo this is now redundant with ActionHistory.reportLineages.
      */
     val sources: List<Source>
```

Comment:
This has not been touched, but I think it can be removed, once the full and glorious lineage tracking is ready.   (Its also buggy, because as in-memory copies of copies of reports occur, the original sources get lost.